src/test/java/com/hocs/test/pages/decs/Workstacks.java

```diff
@@ -480,6 +480,11 @@ public void assertVisibilityOfCaseReference(boolean trueFalse) {
         assertThat(isElementDisplayed(thisReference), is(trueFalse));
     }
 
+    public void assertVisibilityOfSpecificCaseReference(boolean trueFalse, String caseRefNumber) {
+        WebElementFacade thisReference = findBy("//a[text()='" + caseRefNumber + "']");
+        assertThat(isElementDisplayed(thisReference), is(trueFalse));
+    }
+
```

Comment:
Good addition. Could you amend assertVisibilityOfCaseReference method to be called assertVisibilityOfCurrentCaseReference and have it call this new method? 
That way I think we will avoid confusion of what the old method does, and it will be easier to maintain methods if page elements change in future.